api/src/main/java/org/openmrs/module/idgen/service/db/HibernateIdentifierSourceDAO.java

```diff
@@ -248,6 +248,16 @@ public IdentifierSource getIdentifierSourceByUuid(String uuid) {
         criteria.add(Restrictions.eq("uuid", uuid));
         return (IdentifierSource) criteria.uniqueResult();
     }
+    
+    /**
```

Comment:
same here

Reply:
Yes this specific file is very well commented.